<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowMainContainer</class>
 <widget class="QMainWindow" name="WindowMainContainer">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>773</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Employee Database Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>851</width>
      <height>671</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab:selected 
{
    color: white;
    background: rgb(245, 102, 37); 
border: 1px solid black;
   border-radius: 5px;
padding: 0 10;
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_my_edi">
     <attribute name="title">
      <string>My EDI</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>260</y>
        <width>304</width>
        <height>173</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="gridVisible">
       <bool>true</bool>
      </property>
      <property name="navigationBarVisible">
       <bool>true</bool>
      </property>
      <property name="dateEditEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_control">
     <attribute name="title">
      <string>Control</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_search">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>13</y>
        <width>201</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>75</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>75</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>75</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>75</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Salary</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>75</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>SSN</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>75</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Position</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>75</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addEmployee_2">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>41</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Search Employee</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_vieweditemployee">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>40</y>
        <width>161</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>View/Edit Employee</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>831</width>
        <height>551</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>815</width>
         <height>549</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_addEmployee_5">
          <property name="text">
           <string>New Team</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Company Employees</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QTableWidget" name="tableWidget_db">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>EmployeeID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Last Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>First Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Salary</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>SSN</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Position</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Username</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Password</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Team</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_addEmployee_4">
          <property name="text">
           <string>Quick Add</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButton_addEmployee_7">
          <property name="text">
           <string>Quick Remove</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Teams</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_addEmployee_8">
          <property name="text">
           <string>View/Edit Team</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="4">
         <widget class="QTableWidget" name="tableWidget_db_2">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>200</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Team ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Team Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Projects Assigned</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Projects Completed</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_addEmployee_6">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>161</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>New Employee</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_analyze">
     <attribute name="title">
      <string>Analyze</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="labelLogo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="logo.qrc">:/new/prefix1/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelEDI">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>EMPLOYEE DATABASE INTERFACE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LogOut">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_welcomeUser">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Logout_image">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="logo.qrc">:/new/prefix2/EDI-LogOut-Button.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>calendarWidget</tabstop>
  <tabstop>lineEdit_search</tabstop>
  <tabstop>pushButton_addEmployee_2</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>radioButton_3</tabstop>
  <tabstop>radioButton_4</tabstop>
  <tabstop>radioButton_5</tabstop>
  <tabstop>radioButton_6</tabstop>
  <tabstop>radioButton_7</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>tableWidget_db</tabstop>
  <tabstop>pushButton_addEmployee_5</tabstop>
  <tabstop>pushButton_addEmployee_8</tabstop>
  <tabstop>pushButton_addEmployee_4</tabstop>
  <tabstop>pushButton_addEmployee_7</tabstop>
  <tabstop>tableWidget_db_2</tabstop>
  <tabstop>pushButton_addEmployee_6</tabstop>
  <tabstop>pushButton_vieweditemployee</tabstop>
  <tabstop>pushButton_LogOut</tabstop>
 </tabstops>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
